graphql_api/actions/measurements.py

```diff
@@ -44,6 +44,28 @@ def measurements_by_ids(
     return measurements
 
 
+def measurements_last_uploaded_by_before_date(
```

Comment:
a bit confused w/ the naming here since we're using start_date in the timestamp, wouldn't these be measurements_before_start_date?

Reply:
Makes sense, renamed.